src/aatoolbox/datasources/ecmwf/realtime_seas5_monthly.py

```diff
@@ -0,0 +1,194 @@
+        # in the global directory instead of the country specific
+        glb_datasource_class = DataSource(
+            "glb", module_base_dir=_MODULE_BASENAME, is_public=False
+        )
+        self._raw_base_dir = glb_datasource_class._raw_base_dir
+
+        if points_mapping:
+            self._points_mapping = points_mapping
+        elif iso3 in _ISO3_POINTS_MAPPING:
+            self._points_mapping = _ISO3_POINTS_MAPPING[iso3]
+        else:
+            raise Exception(
+                "No point mapping given or iso3 not found in default point "
+                "mappings. Input a point mapping or add the iso3 to defaults."
+            )
+        if iso3 in _ISO3_GEOBB_MAPPING:
+            geo_bounding_box = _ISO3_GEOBB_MAPPING[iso3]
+        else:
+            raise Exception(
+                "Iso3 not found in default bounding box map. "
+                "Add the iso3 bounding box to the defaults."
+            )
+        geo_bounding_box.round_coords(round_val=_GRID_RESOLUTION)
+        self._geobb = geo_bounding_box
+
+    def process(self, datavar: str = "fcmean") -> Path:
+        """
+        Process the seasonal forecast by ECMWF that is shared in realtime.
+
+        This data is private and thus is only shared if you have
+        a license agreement with ECMWF.
+        The data lives in _ECMWF_REALTIME_RAW_DIR
+
+        Parameters
+        ----------
+        datavar: str, default = fcmean
+            variable to extract. fcmean contains the monthly mean
+            per ensemble member. em the mean across all ensemble members
+
+        Returns
+        -------
+        Path to processed NetCDF file
+
+        Examples
+        --------
+        >>> (from aatoolbox.datasources.ecmwf.realtime_seas5_monthly
+        ... import EcmwfRealtime)
+        >>> ecmwf_rt=EcmwfRealtime(iso3="mwi")
+        >>> ecmwf_rt.process()
+        """
+        # T4L indicates the seasonal forecast with the monthly mean
+        # see here for a bit more explanation
+        # https://confluence.ecmwf.int/pages/viewpage.action?pageId=111155348
+        # the 1 is for extra security to not accidentally catch
+        # thrash (such as index) files
+        filepath_list = list(self._raw_base_dir.glob("*T4L*1"))
+        # i would think setting concat_dim=["time","step"] makes more sense
+        # but get an error "concat_dims has length 2 but the datasets passed
+        # are nested in a 1-dimensional structure"
+        # it seems to work thought when using concat_dim="time"
+        # but would have to test once we have data from several dates..
+        output_filepath = self._get_processed_path()
+        output_filepath.parent.mkdir(exist_ok=True, parents=True)
+        with xr.open_mfdataset(
+            filepath_list,
+            engine="cfgrib",
+            filter_by_keys={
+                "numberOfPoints": self._points_mapping,
+                "dataType": datavar,
+            },
+            concat_dim=["step"],
```

Comment:
meheh note that now that we have data for 2 publications (so different publication dates and steps) this is not working anymore. If only concatenating by one dimension, we get a repetition since it is not an unique identifier (i.e. the step dimension will contain each value twice). 
When trying to merge by two parameters (i.e. `concat_dim=["step","time"]`) we get the error `ValueError: concat_dims has length 2 but the datasets passed are nested in a 1-dimensional structure`. 
Havent't found a solution yet.. 

@turnerm in case you have a genius idea

Reply:
The classic "write it down and you will find a solution" happened. Solution was to not use "nested" anymore but instead `combine="by_coords"` and add the step and time dimension (as these are only coords and not dimensions in the original data)